test/Feersum.Tests/SpecTests.fs

```diff
@@ -54,9 +54,25 @@ let ``spec tests compile and run`` s =
     let sourcePath = Path.Join(specDir, s)
     let exePath = Path.ChangeExtension(Path.Join(specBin, s), "dll")
     let shouldFail = sourcePath.Contains "fail"
+    let libSourcePath = Path.ChangeExtension(sourcePath, "sld")
+    let references =
+        if File.Exists(libSourcePath) then
+            let libPath = Path.Join(specBin, Path.GetFileNameWithoutExtension(libSourcePath) + "-backing.dll")
+            let libOptions =
+                { Configuration = BuildConfiguration.Debug
+                ; OutputType = Lib
+                ; References = [] }
+            match compileFile libOptions libPath libSourcePath with
+            | [] ->
+                [ libPath ]
+            | diags ->
+                failwithf "Compilation error in backing library: %A" diags
```

Comment:
Maybe we could annotate source files with comments to mark which dependencies they have. Dependencies
could be compiled once in a one-time setup and share them between multiple spec tests.